src/shared/components/upgradeBrowser/upgradeBrowser.css

```diff
@@ -0,0 +1,63 @@
+.layout {
+  position: absolute;
+  top: 25%;
+  left: 20%;
+  border: 6px solid grey;
+  background: #f0f2f2;
+  overflow: auto;
+  border-radius: 4px;
+  outline: none;
+  padding: 20px;
+  width: 59%;
+  height: 43%;
+  font-size: 2em;
+}
+
```

Comment:
![selection_061](https://user-images.githubusercontent.com/8835055/40882722-d2d1f84a-66b8-11e8-8639-e6f9f6a1cd37.png)

On smaller screens such as my laptop (resolution 1366 x 653 px), the modal has a scrollbar in it. I suggest changing the CSS for class `.layout` to the following, to ensure that the modal is centered on the screen without restricting the height:

```
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
```

And removing the `height: 43%`.